scripts/set_icmp_state

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python
```

Comment:
>python [](start = 15, length = 6)

Does it also work on python3?

Reply:
In 201911 no, because click is not installed. On later versions, yes.